plugins/modules/change_request_info.py

```diff
@@ -291,10 +298,15 @@ def main():
                 "number",
                 "query",
                 "change_request_mapping",
+                "sysparm_query",
                 "sysparm_display_value",
             ),
         ),
-        mutually_exclusive=[("sys_id", "query"), ("number", "query")],
+        mutually_exclusive=[
+            ("sys_id", "query"),
+            ("number", "query"),
+            ("sysparm_query", "query"),
+        ],
```

Comment:
```suggestion
    ("sys_id", "sysparm_query"),
    ("number", "sysparm_query")
        ],
```

Reply:
Add `sys_id` and `sysparm_query` as mutually exclusive.